src/java/picard/vcf/LiftoverVcf.java

```diff
@@ -85,6 +85,12 @@
                     "accompanying sequence dictionary (.dict file).")
     public File REFERENCE_SEQUENCE = Defaults.REFERENCE_FASTA;
 
+    @Option(shortName = "WMC", doc = "Warn on missing contig.", optional = true)
```

Comment:
long name should be descriptive: WARN_ON_MISSING_CONTIG.

as opposed to what? what happens if this is false?